### ISSUE TITLE
Unable to import from local path

### ISSUE THREAD

**Issue by RickDB** [japanesemediamanager/jmmserver#460]
**Issue**

Files not importing from local import folder.

*To reproduce*

Rename import folder in JMMServer from UNC location back to local disk / path like for example:

`E:\some\anime\`

run import files from server and will get this error:
`
MainWindow.workerImport_DoWork => Importer.RunImport_NewFiles System.Exception: Import Folder not found 'E:\some\anime'`

Do notice that the extra trailing slash gets trimmed off at some point during import, confirmed that local path is accessible from server and should be able to resolve it.
Possible cause is that Nutzcode filesystem resolver below fails to validate path:

https://github.com/japanesemediamanager/jmmserver/blob/master/JMMServer/Entities/ImportFolder.cs#L68

**Comment by RickDB** [comment]
When remove the trailing slash from import folder it no longer reports this error.

**Comment by maxpiva** [comment]
Will check for trailing hash in the resolve code in nutzcode, if found
remove...

**Comment by bigretromike** [comment]
Wasn't this also reported by @hidden4003 I could be wrong about person that reported it but I cleary remamber that something mention same. It saw related to driver path like X:\ and the removing tailing would remove it and then jmm would try to check for X:anime insted of X:\anime

**Comment by da3dsoul** [comment]
Can someone who experienced this verify if it is fixed

**Comment by revam** [comment]
I don't think this is an issue anymore. But if it still is, then please open a new issue with a newer stacktrace/log-snip, instead of re-opening this issue.
